ibm/resource_ibm_kms_key_policies.go

```diff
@@ -204,7 +204,7 @@ func resourceIBMKmsKeyPolicyCreate(context context.Context, d *schema.ResourceDa
 		return diag.Errorf("Could not create policies: %s", err)
 	}
 	d.SetId(key.CRN)
-	return resourceIBMKmsKeyPolicyUpdate(context, d, meta)
```

Comment:
Why is it changed to Read?

Reply:
Actually it was by mistake added to update after create..IT should go to read